recipe/meta.yaml

```diff
@@ -1,34 +1,41 @@
-{% set name = "typeguard" %}
 {% set version = "4.1.5" %}
+{% set min_python = "3.8" %}
```

Comment:
this validates the upstream `python-requires` hasn't changed

Reply:
Validates, how?